docs/dasharo-menu-docs/device-manager.md

```diff
@@ -0,0 +1,192 @@
+key which is used to revert to factory default mode. The PSID can be typically
+found on the sticker present on the disk.
+
+Besides the feature enablement and factory restore options there is an action
+interactive entry on the main TCG Drive Management page:
+
+![](/images/menus/tcg_opal3.jpeg){ class="center" }
+
+It is used to perform TCG OPAL specific operations when the security feature is
+enabled. Enable/disable BlockSID will set/unset "freeze locking" of the drive
+features as requested by OS. One may also enable or disable physical presence
+(firmware popup confirmation of actions requested by OS) when switching
+BlockSID.
+
+To set up the password enter the desired disk submenu and enable the feature by
+selecting the checkbox. Note that not all disks support the security feature of
+TCG OPAL. Reset is required to apply changes. After reset you will be prompted
+to set admin password. When the admin password is set, more options become
+available for the disk:
+
+![](/images/menus/tcg_opal3.jpeg){ class="center" }
+
+All below options require reset to apply changes.
+
+* `Update Diver Admin Password` - to update current admin password. Prompt
+  window appears after reset to enter current and new password
+* `Set Driver User Password` - is used to set an optional user password.
+* `Secure Erase User Data` - performs secure erase of the data after reset.
+* `Admin Revert to factory default and Disable` - disables the admin password
+  (prompt will no longer appear during boot o unlock the drive) and the
+  security feature after reset. Optionally one may select to keep or discard
+  user data.
+* `PSID Revert to factory default and Disable` - in case admin password has
+  been forgotten PSID can be used to disable the security feature and admin
+  password after reset. This will also erase user data.
+* `Disable User` - disables the user functionality (and so the user password)
+  after reset.
+
+For more details about TCG OPAL please refer to
+[TCG Storage Specifications and Key Management](https://trustedcomputinggroup.org/resource/tcg-storage-specifications-and-key-management/).
+
+## Driver Health Manager
+
+This submenu is informational only and displays the status of drivers that
+install UEFI driver health protocol. Typically it is installed by 3rdparty
+applications such as OptionROMs on graphics devices or network controllers for
+example.
+
+![](/images/menus/health_mgr.jpeg){ class="center" }
+
+## Secure Boot Configuration
+
+This submenu allows to configure UEFI Secure Boot functionality. By default
+Dasharo firmware released after October 2022 has Secure Boot disabled by
+default with no keys and certificates provisioned.
+
+![](/images/menus/secure_boot.jpeg){ class="center" }
+
+Option to enable Secure Boot (`Attempt Secure Boot`) will be grayed out as long
+as there are missing keys and certificates in UEFI variables. In order to
+provision default keys embedded into firmware image, select the
```

Comment:
Maybe  specify that these are the usual Microsoft keys for booting Windows and shim.

Reply:
Added

Reply:
Thanks